tests/unit/customizations/s3/test_utils.py

```diff
@@ -653,6 +653,28 @@ def test_upload_part_copy(self):
              'SSECustomerKey': 'my-sse-c-key'})
 
 
+class TestRequestParamsMapperChecksumAlgorithm:
```

Comment:
Let's also add test cases for:
* Copy object
* Empty `cli_params` and assert `ChecksumAlgorithm` isn't present

Reply:
Good points, resolving in next iteration